Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM alpine
+FROM alpine:edge AS build
+WORKDIR /go/src/github.com/prebid/prebid-server/
+RUN apk add -U --no-cache go git fftw-dev musl-dev dep
+ENV GOPATH /go
+COPY ./ ./
+RUN dep ensure
+RUN go build .
+
+
+FROM alpine:edge AS release
```

Comment:
`edge` is the development branch of alpine--I don't think we'd want to use this for release images.